tests/test_general_reduction_ops.py

```diff
@@ -123,6 +123,20 @@ def test_accuracy_max_without_dim(shape, dtype):
     gems_assert_equal(res_out, ref_out)
 
 
+@pytest.mark.max
+@pytest.mark.parametrize("shape", REDUCTION_SHAPES)
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES)
+def test_accuracy_max_without_dim_uncontiguous(shape, dtype):
+    inp = torch.randn(shape, dtype=dtype, device="cuda")[::2,::2]
```

Comment:
code-format-check / pre-commit failed here.
Please change it to `[::2, ::2]`. Thanks.

Reply:
done